### DISCUSSION TITLE
Getting rejected 3 times for student developer pack

### DISCUSSION THREAD

**Discussion by aarjn** [community/community#63843] [GitHub Education]
### Hi everyone,

I have been applying for student developer pack for past  3 days and my application is getting rejected. How can contact someone to resolve this issue. Everything my naming in bio and ID cards are same, i have uploaded my ID card with date

**Comment by safnimj** [comment]
Can you please mention the issue that mentioned in the rejection Email

**Reply by aarjn** [reply]
You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.


I have made the changes. Still same
